crates/imposters/src/render.rs

```diff
@@ -489,8 +489,8 @@ pub struct ImposterSpecManager<'w, 's> {
     plugin: Res<'w, DclImposterPlugin>,
 }
 
-const MAX_ASSET_LOADS: usize = 40;
-const MAX_ASSET_DOWNLOADS: usize = 10;
+const MAX_ASSET_LOADS: usize = 5;
+const MAX_ASSET_DOWNLOADS: usize = 5;
```

Comment:
Can we change the values based on the Platform? (Desktop & Web)

Reply:
i'm going to revert this. i thought it was causing an issue but i found the real cause